src/App.tsx

```diff
@@ -1,16 +1,69 @@
-import { useState } from "react";
+import { useEffect, useState } from "react";
 import "./assets/scss/styles.scss";
 import TodoForm from "./components/todo/TodoForm";
 import TodoList from "./components/todo/TodoList";
 import { TodoModal } from "./models";
+import { UserModal } from "./models/user";
 import { staticTodo } from "./mock-data/todo";
+import Login from "./components/shared/Login";
+import { gapi } from "gapi-script";
+
+const clientId =
```

Comment:
Keep the sensitive information in env file, out of repository.

Reply:
.env file created and placed clientId there